SpywareFilter/sections/specific.txt

```diff
@@ -52,7 +52,9 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com|bolgehastanesi.com|ewtc.de
 !
+!#if ( !adguard_app_windows )
 ||search.naver.com/p/crd/rd
+!#endif
```

Comment:
The same issue occurs on Android (I suppose that the same is with AdGuard for Mac). If I'm not wrong, it's not blocked by extensions, because extensions do not block main document.
So, if it's used only as "document" then I think that it would be better to remove this rule.